src/kerberos/mod.rs

```diff
@@ -1006,18 +1006,77 @@ impl SspiEx for Kerberos {
     }
 }
 
-#[cfg(test)]
-mod tests {
+#[cfg(any(feature = "test_data", test))]
+pub fn test_client() -> Kerberos {
```

Comment:
nit: Maybe it would be better to place test_client and test_server inside a module instead?:
```
#[cfg(any(feature = "test_data", test))]
pub mod test_data {
    pub fn fake_client() -> Kerberos { ... }
    pub fn fake_server() -> Kerberos { ... }
}
```

Reply:
agreed. done :heavy_check_mark: